python/ray/serve/_private/replica_scheduler/pow_2_scheduler.py

```diff
@@ -517,7 +517,7 @@ async def _probe_queue_lens(
                 # If we get a RayActorError, it means the replica actor has died. This
                 # is not recoverable (the controller will start a new replica in its
                 # place), so we should no longer consider it for requests.
-                if isinstance(t.exception(), RayActorError):
+                if isinstance(t.exception(), ActorDiedError):
```

Comment:
can you please fix the above comment to reflect the new behavior and why we are explicitly _not_ catching generic RayActorError

Reply:
I updated the comment.